tests/terratest/functions/getClusterID.go

```diff
@@ -0,0 +1,53 @@
+package functions
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+func GetClusterID(hostURL string, clusterName string, token string) string {
```

Comment:
This goes for all the functions, you have implemented in the functions package. Why is there a need to do direct rest calls to these resources, rather than using the generated clients? Are you using a token to access these resources in a rancher instance? I mean if you use the clients there would no need for this package to be honest.

Reply:
If the functions are redundant they don't need to be added.  I'm still getting up to speed with the existing automation framework and have not utilized it yet.  I did not realize the existing framework could be used with these tests, if that's what you're suggesting.  This was just my approach and yes, i'm using a token in the GET headers to authenticate the request.  If we choose to leave out the functions, the tests will need to be refactored.

Reply:
Awesome I shared a link in my other comment, or you can use the framework, you would just import the package. I'm leaning on using the framework do these calls. The framework can be imported to this no problem.

Reply:
There are some functions ending with TF, that would still need to be included in the package, so there is still a need for the package, but I agree, all the api calls could/should be removed

Reply:
For sure, just the rest calls not the terraform stuff